src/test/java/stats/ResampleTest.java

```diff
@@ -9,6 +9,18 @@
 public class ResampleTest {
```

Comment:
I don't quite get the structure of this test. maybe you can elaborate. I get the basics and what it does but getNumericSample baffles me a little.

Reply:
It simply checks whether the ```resample``` creates the sample of the right size. Perhaps I need to improve the comments...Is that better now?

Reply:
much better. thanks!